libs/application/templates/estate/src/lib/messages.ts

```diff
@@ -0,0 +1,655 @@
+    description: '',
+  },
+  personalInfoProviderTitle: {
+    id: 'es.application:personalInfoProviderTitle',
+    defaultMessage: 'Persónuupplýsingar um þig',
+    description: '',
+  },
+  personalInfoProviderSubtitle: {
+    id: 'es.application:personalInfoProviderSubtitle',
+    defaultMessage: 'Upplýsingar frá Þjóðskrá um kennitölu og lögheimili.',
+    description: '',
+  },
+  settingsInfoProviderTitle: {
+    id: 'es.application:settingsInfoProviderTitle',
+    defaultMessage: 'Stillingar frá Ísland.is',
+    description: '',
+  },
+  settingsInfoProviderSubtitle: {
+    id: 'es.application:settingsInfoProviderSubtitle',
+    defaultMessage: 'Persónustillingar (sími og netfang) þínar frá Ísland.is.',
+    description: '',
+  },
+
+  // The deceased
+  theDeceased: {
+    id: 'es.application:theDeceased',
+    defaultMessage: 'Hinn látni',
+    description: '',
+  },
+  nameOfTheDeceased: {
+    id: 'es.application:nameOfTheDeceased',
+    defaultMessage: 'Nafn hins látna',
+    description: '',
+  },
+  deathDate: {
+    id: 'es.application:deathDate',
+    defaultMessage: 'Dánardagur',
+    description: '',
+  },
+  deathDateNotRegistered: {
+    id: 'es.application:deathDateNotRegistered',
+    defaultMessage: 'Dánardagur ekki skráður',
+    description: '',
+  },
+
+  // Applicant
+  announcer: {
+    id: 'es.application:announcer',
+    defaultMessage: 'Yfirlýsandi eignaleysis',
+    description: '',
+  },
+  applicantsInfoSubtitle: {
+    id: 'es.application:applicantsInfoSubtitle',
+    defaultMessage:
+      'Vinsamlegast farðu yfir upplýsingarnar og athugaðu hvort þær séu réttar.',
+    description: '',
+  },
+  name: {
+    id: 'es.application:name',
+    defaultMessage: 'Nafn',
+    description: '',
+  },
+  nationalId: {
+    id: 'es.application:nationalId',
+    defaultMessage: 'Kennitala',
+    description: '',
+  },
+  address: {
+    id: 'es.application:address',
+    defaultMessage: 'Lögheimili',
+    description: '',
+  },
+  phone: {
+    id: 'es.application:phone',
+    defaultMessage: 'Símanúmer',
+    description: '',
+  },
+  email: {
+    id: 'es.application:email',
+    defaultMessage: 'Netfang',
+    description: '',
+  },
+
+  // Estate members, assets, vehicles
+  estateMembersTitle: {
+    id: 'es.application:estateMembersTitle',
+    defaultMessage: 'Erfingjar og erfðaskrá',
+    description: '',
+  },
+  estateMembersSubtitle: {
+    id: 'es.application:estateMembersSubtitle',
+    defaultMessage:
+      'Vinsamlegast farðu yfir upplýsingarnar og athugaðu hvort þær séu réttar.',
+    description: '',
+  },
+  estateMembers: {
+    id: 'es.application:estateMembers',
+    defaultMessage: 'Erfingjar',
+    description: '',
+  },
+  estateMembersHeaderDescription: {
+    id: 'es.application:estateMembersHeaderDescription',
+    defaultMessage: 'Vantar uppl hér',
```

Comment:
```suggestion
    defaultMessage: 'Vantar uppl. hér',
```

Reply:
just a temp text, will replace once we get proper texts

Reply:
:+1: